redfish-core/lib/log_services.hpp

```diff
@@ -841,6 +842,21 @@ inline void deleteDumpEntry(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
                 messages::resourceNotFound(asyncResp->res, "LogEntry", entryID);
                 return;
             }
+
+            const sd_bus_error* dbusError = msg.get_error();
+            if (dbusError == nullptr)
+            {
+                messages::internalError(asyncResp->res);
+                return;
+            }
+
+            if (strcmp(dbusError->name, "xyz.openbmc_project.Common.Error."
+                                        "Unavailable") == 0)
+            {
+                messages::serviceTemporarilyUnavailable(asyncResp->res, "1");
```

Comment:
1 second seems like not much time? we sure we will be done in 1 second ?

Reply:
1 second is  enough and HMC anyways re-try.

Reply:
IDK, 1 seconds doesn't seem like a enough.. We can change later if we need